packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -162,14 +162,18 @@ const extractContent = (s: string): string => {
 };
 
 const messageContentAriaText = (props: ChatMessageContentProps): string | undefined => {
-  return props.message.content
+  // Strip all html tags from the content for aria.
+  const regexForStrippingHtml = /<[^>]*>|<script/gm;
+  const htmlStrippedContent = props.message.content?.replaceAll(regexForStrippingHtml, '');
```

Comment:
I'm guessing we are stripping out any possible HTML here since when images or responses were sent it would read out the tags?

Reply:
Mentions are encoded in a <msft-mention> tag, which we don't want read out. More generally, we don't want tags read too